app/views/layouts/application.html.erb

```diff
@@ -8,12 +8,26 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+
+    <%= stylesheet_link_tag "sticky-footer-navbar" %>
   </head>
 
   <body>
     <header class='container'>
       <%= render 'layouts/navbar' %>
     </header>
+
     <%= yield %>
+
+    <footer class="footer container-fluid">
+      <%= render 'layouts/footer' %>
+    </footer>
+
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" crossorigin="anonymous">
+    </script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.14.7/dist/umd/popper.min.js" crossorigin="anonymous">
+    </script>
```

Comment:
I think you may be able to install them using NPM:

```shell
npm install --save jquery popper.js
```

and maybe you won't need to add them manually here :)

Reply:
I install them and that doesn't work :(

Reply:
I try use importmap as well, not work, but in the importmap I'm not sure, maybe needs some more to set like javascript controllers?